ext/digest/extconf.rb

```diff
@@ -8,4 +8,6 @@
   "digest.h" => "$(HDRDIR)"
 }
 
+have_func("rb_ext_ractor_safe")
```

Comment:
`HAVE_RB_EXT_RACTOR_SAFE` is already defined on 3.0
b29c0fadeca7bd7d00d9fe17345fe88bf968312f

Reply:
Removed.